src/v1alpha3/echo_client.ts

```diff
@@ -0,0 +1,233 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as gax from 'google-gax';
+import * as path from 'path';
+import * as protobuf from 'protobufjs';
+
+import * as packageJson from '../../package.json';
+import * as protos from '../../pbjs-genfiles/protos';
+import * as gapicConfig from './echo_client_config.json';
+
+const version = packageJson.version;
+
+export interface ClientOptions extends gax.GrpcClientOptions,
+                                       gax.GoogleAuthOptions,
+                                       gax.ClientStubOptions {
+  libName?: string;
+  libVersion?: string;
+  clientConfig?: gax.ClientConfig;
+}
+
+interface Descriptors {
+  page: {[name: string]: gax.PageDescriptor};
+  stream: {[name: string]: gax.StreamDescriptor};
+  longrunning: {[name: string]: gax.LongrunningDescriptor};
+}
+
+export interface Callback<ResponseObject> {
+  (err: Error|null|undefined, value?: ResponseObject|null): void;
+}
+
+export interface PaginationCallback<
+    RequestObject, ResponseObject, ResponseType> {
+  (err: Error|null, values?: ResponseType[], nextPageRequest?: RequestObject,
+   rawResponse?: ResponseObject): void;
+}
+
+export interface PaginationResponse<
+    RequestObject, ResponseObject, ResponseType> {
+  values?: ResponseType[];
+  nextPageRequest?: RequestObject;
+  rawResponse?: ResponseObject;
+}
+
+export class EchoClient {
+  _descriptors: Descriptors = {page: {}, stream: {}, longrunning: {}};
+  _innerApiCalls: {[name: string]: Function};
+  auth: gax.GoogleAuth;
+
+  constructor(opts: ClientOptions) {
```

Comment:
Just to have the discussion - are we dropping traditional jsdoc annotations?

Reply:
I didn't bother adding annotations to make the code readable while we're in this phase. I'll need to add _some_ annotations anyway.